code/src/components/Image.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+
+export const Image = ({ src, alt, className, ...props }) => {
+  return (
+    <ImageWrapper>
+      <CircularImage src={src} alt={alt} />
+    </ImageWrapper>
```

Comment:
Also here, to make this even more reusable you could have an Image where you can pass in a prop called circular which will take care of changing the border radius depending on the prop? This works fine though, just wanted to give you some examples.